app/src/main/java/com/example/movietrackerapp/model/remote/service/response/MovieResponse.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.movietrackerapp.model.remote.service.response
+
+import com.google.gson.annotations.SerializedName
+
+data class MovieResponse (
```

Comment:
Looks like this is autogenerated by a tool (?), if so , can you hand me the tool you used please ? also , reformat the code  for styling.
```
data class MovieResponse (
    @SerializedName("page") var page: Int? = null,
....
```